user-data-stream_CN.md

```diff
@@ -31,7 +31,7 @@ NONE
 }
 ```
 
-## 演唱lisenKey有效期
+## 延长lisenKey有效期
```

Comment:
Shouldn't it be `listenKey`?

Reply:
😍

Reply:
https://github.com/binance-exchange/binance-official-api-docs/pull/102